src/components/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,17 @@
+// import "./LandingPage.scss"
+
+const LandingPage = () => {
+  return(
+    <div>
+      <img></img>
+      <p>We will accept entries for the 2023 Turing Contest beginning April 1st, 2022. The entrance fee is $10 per song. Contestants may submit an original song. We will post submission details later on.
+
+      Important submission dates:
+
+      April 22nd, 2023: All submissions must be postmarked on or before.
+      May 4th, 2023: We will notify all finalists and alternates by this date. Watch for the public posting of Troubadour finalists around this time.</p>
+    </div>
+  )
+}
+
+export default LandingPage
```

Comment:
Looks great! Thank you for putting this together!

Reply:
Looks really good so far!